packages/office-ui-fabric-react/src/components/Button/Button.tsx

```diff
@@ -19,7 +25,7 @@ export interface IButtonState {
 
 export class Button extends BaseComponent<IButtonProps, IButtonState> implements IButton {
```

Comment:
We should have a `BaseButton` or `ButtonBase` separate file. We could keep Button intact as is, but then deprecate it in favor of the subclasses. `PrimaryButton`, `NormalButton`, `CommandButton`, etc.